### DISCUSSION TITLE
Not being able to create Virtual machines in Cockpit at ubuntu server 22.04

### DISCUSSION THREAD

**Discussion by tai5555** [cockpit-project/cockpit#18259] [Q&A]
Hi I am not being able to create virtual machines in my server at ubuntu 22.04. The error is below kindly do guide me how to resolve it!

error: Failed to destroy domain 'WAF' error: Requested operation is not valid: domain is not running ERROR internal error: qemu unexpectedly closed the monitor: 2023-01-28T09:16:55.869674Z qemu-system-x86_64: -blockdev {"driver":"file","filename":"/home/staging/ubuntu-22.04.1-live-server-amd64.iso","node-name":"libvirt-1-storage","auto-read-only":true,"discard":"unmap"}: Could not open '/home/staging/ubuntu-22.04.1-live-server-amd64.iso': Permission denied Domain installation does not appear to have been successful. If it was, you can restart your domain by running: virsh --connect qemu:///system start WAF otherwise, please restart your installation. Command '['virt-install', '--connect', 'qemu:///system', '--quiet', '--name', 'WAF', '--os-variant', 'ubuntu22.04', '--memory', 'memory=16384,maxmemory=16384', '--check', 'path_in_use=off', '--wait', '-1', '--noautoconsole', '--disk', 'path=/var/lib/libvirt/images/WAF.qcow2,device=disk,bus=virtio,readonly=off,shareable=off,discard=unmap,driver_name=qemu,driver_type=qcow2', '--network', 'bridge=br0,mac=52:54:00:e0:ad:a6,m
![error_cockpit](https://user-images.githubusercontent.com/97503378/215258171-54ad981d-3f39-4e6c-a495-cb3171f8bef0.PNG)
odel=virtio,link_state=up', '--graphics', 'type=vnc,listen=127.0.0.1,port=-1', '--graphics', 'type=spice,listen=127.0.0.1', '--cdrom', '/home/staging/ubuntu-22.04.1-live-server-amd64.iso', '--vcpus', 'vcpus=2,maxvcpus=2', '--boot', '']' returned non-zero exit status 1.

> 

>

**Comment by tai5555** [comment]
Issue is resolved

**Reply by Logikoz** [reply]
how? @tai5555

**Reply by KerwinKoo** [reply]
how?

**Reply by tomasmatus** [reply]
Can't say for sure what is going on but I suppose you need to check your permissions on the ISO.
`Could not open '/home/staging/ubuntu-22.04.1-live-server-amd64.iso': Permission denied`
